samples/msal-node-auth-code/index.js

```diff
@@ -4,12 +4,36 @@
  */
 const express = require("express");
 const msal = require('@azure/msal-node');
-const myLocalCache = require("./data/cache");
 const fs = require("fs");
 
 const SERVER_PORT = process.env.PORT || 3000;
 
 // initialize msal public client application
+const cachePlugin = {
```

Comment:
Example of what developer implementing their own persistence might look like but ideally the extension lib will pass in these funcs